blockchain/fullblocks_test.go

```diff
@@ -146,6 +146,62 @@ func TestFullBlocks(t *testing.T) {
 	}
 	defer teardownFunc()
 
+	testBlockDisconnectExpectUTXO := func(item fullblocktests.BlockDisconnectExpectUTXO) {
+		expectedCallBack := func(notification *blockchain.Notification) {
+			switch notification.Type {
+
+			case blockchain.NTBlockDisconnected:
+				block, ok := notification.Data.(*btcutil.Block)
+				if !ok {
+					t.Fatalf("expected a block")
+				}
+
+				if block.Hash().IsEqual(&item.BlockHash) {
```

Comment:
nit: invert condition and `return` to continue non-indented below?

Reply:
Applied in the latest push